scripts/main.ps1

```diff
@@ -376,13 +373,14 @@ $sync["SearchBar"].Add_TextChanged({
 
     $activeApplications = @()
 
+    $textToSearch = $sync.SearchBar.Text.ToLower()
+
```

Comment:
just wanna say well done on this, as this lower case conversion was done on every single checkbox.. which's.. lets sat not that efficient 😅 
Good & simple optimization @MyDrift-user 👏